src/autolabel/labeler.py

```diff
@@ -66,17 +66,21 @@ def run(
             output_name: custom name of output CSV file
             start_index: skips annotating [0, start_index)
         """
+        dataset_loader = DatasetLoader(dataset, self.config, max_items, start_index)
 
         self.db.initialize()
         self.dataset = self.db.initialize_dataset(
-            dataset, self.config, start_index, max_items
+            dataset_loader.dat, self.config, start_index, max_items
```

Comment:
can we access the dataframe with a helper method in dataset loader (vs directly accessing it here) ? for readability

Reply:
Can you give an example of what you want it to look like instead?

Reply:
i was thinking just having a `def as_datafame()` method in the data loader that returns the internal pandas dataframe object, instead of the labeling agent accessing it directly